packages/notifications/src/styled/global-alert/StyledGlobalAlertClose.ts

```diff
@@ -6,54 +6,66 @@
+
   let hoverBackgroundColor;
   let hoverForegroundColor;
   let activeBackgroundColor;
   let activeForegroundColor;
-  let focusColor;
-
-  switch (props.alertType) {
-    case 'success':
-      hoverBackgroundColor = getColorV8('successHue', 100, props.theme, 0.08);
-      hoverForegroundColor = props.theme.palette.white;
-      activeBackgroundColor = getColorV8('successHue', 100, props.theme, 0.2);
-      activeForegroundColor = props.theme.palette.white;
-      focusColor = 'successHue';
-      break;
-
-    case 'error':
-      hoverBackgroundColor = getColorV8('dangerHue', 100, props.theme, 0.08);
-      hoverForegroundColor = props.theme.palette.white;
-      activeBackgroundColor = getColorV8('dangerHue', 100, props.theme, 0.2);
-      activeForegroundColor = props.theme.palette.white;
-      focusColor = 'dangerHue';
-      break;
-
-    case 'warning':
-      hoverBackgroundColor = getColorV8('warningHue', 800, props.theme, 0.08);
-      hoverForegroundColor = getColorV8('warningHue', 900, props.theme);
-      activeBackgroundColor = getColorV8('warningHue', 800, props.theme, 0.2);
-      activeForegroundColor = getColorV8('warningHue', 1000, props.theme);
-      focusColor = 'warningHue';
-      break;
-
-    case 'info':
-      hoverBackgroundColor = getColorV8('primaryHue', 700, props.theme, 0.08);
-      hoverForegroundColor = getColorV8('primaryHue', 800, props.theme);
-      activeBackgroundColor = getColorV8('primaryHue', 700, props.theme, 0.2);
-      activeForegroundColor = getColorV8('primaryHue', 900, props.theme);
-      focusColor = 'primaryHue';
-      break;
+  let focusVariable;
+
+  if (['success', 'error'].includes($alertType)) {
+    const variable = $alertType === 'success' ? 'background.success' : 'background.danger';
+    focusVariable =
+      $alertType === 'success' ? 'foreground.successEmphasis' : 'foreground.dangerEmphasis';
+
+    hoverBackgroundColor = getColor({ variable, theme, transparency: theme.opacity[100] });
+    hoverForegroundColor = theme.palette.white;
+    activeBackgroundColor = getColor({ variable, theme, transparency: theme.opacity[200] });
+    activeForegroundColor = theme.palette.white;
+  } else if ($alertType === 'warning') {
+    const bgVariable = 'background.warningEmphasis';
+    const foregroundVariable = 'foreground.warningEmphasis';
+    focusVariable = 'foreground.warning';
+
+    hoverBackgroundColor = getColor({
+      variable: bgVariable,
+      transparency: theme.opacity[100],
+      theme
+    });
+    hoverForegroundColor = getColor({
+      variable: foregroundVariable,
+      light: { offset: 200 },
+      theme
+    });
+    activeBackgroundColor = getColor({
+      variable: bgVariable,
+      transparency: theme.opacity[200],
+      theme
+    });
+    activeForegroundColor = getColor({
+      variable: foregroundVariable,
+      light: { offset: 300 },
+      theme
+    });
+  } else {
+    focusVariable = 'foreground.primary';
   }
```

Comment:
ditto re: ternaries .. the previous `switch` feels more readable/straightforward to process.